source/PlayerInfoPanel.h

```diff
@@ -19,6 +19,7 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include "Panel.h"
 
 #include "ClickZone.h"
+#include "Color.h"
```

Comment:
Is this used by this file?

Reply:
I get red squigglies without it. The tooltip needs a foreground and background colour. I only use `Color` as a variable. I could inline it or perhaps declare that a class `Color` exists as is done in some header files. Which would be your preference?

Reply:
I I've just realised this was the header file you were looking at. I thought it was the cpp file; the function did originally have five arguments, with fg/bg colour being two of them — I copied it from elsewhere, but since it only gets called from one place, there's no need to have so many args.